ui/src/pages/frontsite/PatronProfile/PatronMetadata/PatronMetadata.js

```diff
@@ -1,18 +1,13 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Header } from 'semantic-ui-react';
 
 export default function PatronMetadata(props) {
   return (
     <>
-      <Header as="h2">Welcome {props.user.username}!</Header>
+      <Header as="h2">Your activity</Header>
       <Header.Subheader>
-        You can check your library activity on this page
+        Here you can find your library activity.
```

Comment:
```suggestion
        This is your library activity.
```

Reply:
Not sure about this, any other opinion from any one else? I will actually remove the sentence.

Reply:
Removed